deco-sites/otica-isabela
# add stopwatch into product-card

## What is this contribution about?

Add stop watch within Product Card to show promotional time remaining.

## How to test it?

Go to HomePage or any page that shows shelves with Promotional Products and see in the top of the Card the Stopwatch..
